app/base/static/profile_pictures/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+*/
+!.gitignore
```

Comment:
in a project we only need to have just 1 `.gitingore` file on the base of the project
remove it from here